### PR TITLE
Feature/ds/update lead channel

### PR DESCRIPTION
**Pull request by Deeptrentium**

### CODE REVIEW

File: components/organisms/bottom-sheet-Navigator-Screen/LeadChannelList.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    (state: RootState) => state.leads.leadsDetail,
+  );
+  const getLeadSChannelList = async () => {
+    await dispatch(leadChannelListAction());
+  };
+  const onLayout = useCallback(() => {
+    changeSnapPoints(['50%', '90%']);
+  }, []);
+
+  const handleItemPress = async (ItemId: number) => {
```
Comment (sagark1510) [inline components/organisms/bottom-sheet-Navigator-Screen/LeadChannelList.tsx:44]: ItemId ??

bhai, how many times I have to point out such silly issues?

File: components/organisms/bottom-sheet-Navigator-Screen/LeadChannelList.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  }, []);
+
+  const renderModifyLeadOption = ({
+    item,
+    index,
+  }: {
+    item: LeadStageListItemProps;
+    index: number;
+  }) => {
+    const isStatusSelected = !!(leadsDetail.leadChannelId === item.id);
```
Comment (sagark1510) [inline components/organisms/bottom-sheet-Navigator-Screen/LeadChannelList.tsx:75]: why do we need !! ? === will always return boolean only

File: components/organisms/bottom-sheet-Navigator-Screen/modifyLeadOption.tsx [hunk 3]
```diff
@@ -42,7 +44,7 @@ const ModifyLeadOption: React.FC<ModifyLeadOptionProps> = ({
     if (optionType === ScreenOptionType.DASHBOARD) {
       changeSnapPoints(['31%', '90%']);
     } else if (optionType === ScreenOptionType.LEAD) {
-      changeSnapPoints(['31%', '90%']);
+      changeSnapPoints(['41%', '90%']);
```
Comment (sagark1510) [inline components/organisms/bottom-sheet-Navigator-Screen/modifyLeadOption.tsx:47]: make sure you test this % values in different dimensions and different devices to make sure it won't break.

**Review by sagark1510 — CHANGES_REQUESTED** [review]

File: components/organisms/bottom-sheet-Navigator-Screen/modifyLeadOption.tsx (hunk 3 above)
Comment (Deeptrentium) [reply components/organisms/bottom-sheet-Navigator-Screen/modifyLeadOption.tsx:47]: yes sir

**Review by sagark1510 — APPROVED** [review]

**Pull request merged**